<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDF 2D Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QHBoxLayout" name="inputLayout">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="paramsGroup">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QFormLayout" name="paramsLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="outputWidthLabel">
         <property name="text">
          <string>Output Width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="outputWidthSpin"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="outputHeightLabel">
         <property name="text">
          <string>Output Height</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="outputHeightSpin"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="thresholdLabel">
         <property name="text">
          <string>Threshold</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="thresholdSpin"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="maxDistanceLabel">
         <property name="text">
          <string>Max Distance</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxDistanceSpin"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="actionsGroup">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QHBoxLayout" name="actionsLayout">
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate SDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save PNG</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="previewGroup">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QHBoxLayout" name="previewLayout">
       <item>
        <layout class="QVBoxLayout" name="originalLayout">
         <item>
          <widget class="QLabel" name="originalTitle">
           <property name="text">
            <string>Original</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="originalLabel">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="sdfLayout">
         <item>
          <widget class="QLabel" name="sdfTitle">
           <property name="text">
            <string>SDF</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sdfLabel">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
